### ISSUE TITLE
Error importing

### ISSUE THREAD

**Issue by phazejeff** [clbrge/svelte-web3#14]
I have this in my App.svelte file:
```
<script>
	import { defaultChainStore, web3, selectedAccount, connected, chainData } from 'svelte-web3'

	defaultChainStore.setBrowserProvider()
</script>
```

I get this error:
```
[!] Error: 'default' is not exported by node_modules\svelte-web3\src\chains.json, imported by node_modules\svelte-web3\src\web3-store.js
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
node_modules\svelte-web3\src\web3-store.js (2:7)
1:
2: import chains from './chains.json'
          ^
3: import { derived, writable } from 'svelte/store'
Error: 'default' is not exported by node_modules\svelte-web3\src\chains.json, imported by node_modules\svelte-web3\src\web3-store.js
```

**Comment by monomadic** [comment]
I'm also getting this error. The node ecosystem seems to be rife with this kind of inconsistency and vague errors.

**Comment by clbrge** [comment]
@monomadic Yes, because the ecosystem is so big. but we would need more details in your precise context to help you. most probably .json fails to import in your context.

**Comment by jonathanhecl** [comment]
I have the same error.

```
node_modules\svelte-web3\src\web3-store.js (2:7)
1:
2: import chains from './chains.json'
          ^
3: import { derived, writable } from 'svelte/store'
Error: 'default' is not exported by node_modules\svelte-web3\src\chains.json, imported by node_modules\svelte-web3\src\web3-store.js
```

**Comment by connorrothschild** [comment]
@jonathanhecl I fixed this by adding `rollup-plugin-json` to my `rollup.config.js` ([see this question](https://stackoverflow.com/questions/52063652/importing-local-json-in-main-js-in-svelte)). You should be able to add

```js
import json from 'rollup-plugin-json';
``` 

to the top of your config, and then in `plugins: [`, add the json options (I just used the default from [here](https://github.com/rollup/rollup-plugin-json#usage)):

```js
json({
    include: 'node_modules/**',
    exclude: [ 'node_modules/foo/**', 'node_modules/bar/**' ],

    preferConst: true,
    
    indent: '  ',
    
    compact: true, 
   
    namedExports: true
}),
```

**Comment by clbrge** [comment]
Thanks for the information @connorrothschild (I don't use rollup myself anymore) I'll add a note in the README so people know what to do in this case.

**Comment by clbrge** [comment]
@jonathanhecl @jonathanhecl Acually, i just released a new version (2.2.0) which removes the `chains.json`  and replace it with a pure js equivalent.
So `rollup-plugin-json` is not needed anymore.
